DuckDuckGo/UserText.swift

```diff
@@ -1157,8 +1157,7 @@ But if you *do* want a peek under the hood, you can find more information about
     // MARK: -  Dax Onboarding Experiment
     public enum DaxOnboardingExperiment {
         enum Intro {
-            public static let title = NSLocalizedString("onboarding.intro.title", value: "Hi there.", comment: "The title of the onboarding dialog popup")
```

Comment:
I was originally using two labels for the Intro dialog. I changed to one label as it was becoming  more complicated handling multiple flags to animate the two labels. Also, when I was tapping on the dax dialog to test interrupting the animation the second label would appear and then restart animating again 😬